models/articles.model.js

```diff
@@ -1,13 +1,15 @@
 const db = require('../db/connection.js');
 
-exports.selectArticles = () =>
+exports.selectArticles = (topic = '%') =>
```

Comment:
I would expect to see some whitelisting here of valid topics to sort on. You'll need to find a dynamic way to check available topics and if the passed topic is valid.